### PR TITLE
seasonal cookies

### PR DESCRIPTION
**Pull request by robintime**
fgdfgdfgdsDs

**5 flour, 5 sugar, and 5 sprinkles** will make **3 cookies**, the appearances of which change based on the month. If the current month isn't October or December, the recipe will produce any possible appearances from any holiday.

Each cookie has 6 nutriment and 2 sugar because I can't do math properly.

:cl:
 * rscadd: new food. yes i will update the wiki i swear

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -647,6 +647,34 @@
...
+		switch(Holiday)
+			if(XMAS || XMAS_EVE)
+				cookiecutter = pick( list("stocking","tree","snowman","mitt","angel","deer") )
+				icon_state = "[cookiecutter]"
+			if(HALLOWEEN)
+				cookiecutter = pick( list("spider","cat","pumpkin","bat","ghost","hat","frank") )
+				icon_state = "[cookiecutter]"
+			else
+				cookiecutter = pick( list("spider","cat","pumpkin","bat","ghost","hat","frank","stocking","tree","snowman","mitt","angel","deer") )
+				icon_state = "[cookiecutter]"
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: just put this once outside the switch statement

Comment (robintime) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: Thy will hath been done.

**Comment by GreatBigFailure** [comment]
I love them

**Comment by Kurfursten** [comment]
A value pack of more cookie designs if you want them
image

Can send you the DMI via Discord

**Comment by robintime** [comment]
> A value pack of more cookie designs if you want them image
> 
> Can send you the DMI via Discord

holy shit YES

**Comment by SonixApache** [comment]
make better :cl:

**Comment by Eneocho** [comment]
> 
> 
> A value pack of more cookie designs if you want them image
> 
> Can send you the DMI via Discord

Fucking 6 autism awareness cookies.

**Comment by Kurfursten** [comment]
Sonix suggested on Discord, and I second, that you should get Halloween cookies for all of October and Christmas for all of December. You can do this by checking the month instead of the holiday.

**Comment by Servalfang** [comment]
need chicken cookies and a mouse cursor cookie

**Comment by Konater** [comment]
thumbs down it sucks!!!!

**Comment by SonixApache** [comment]
the :cl: 😔

**Pull request merged**